chore: add SECURITY.md (#7733)
web-infra-dev/rspack: SECURITY.md

# Security

> ## Reporting a security bug in Rspack

Reviewer: ```suggestion
## Reporting a vulnerability in Rspack
```

> ## Reporting a vulnerability in Rspack

Reviewer: ```suggestion

## Reporting a vulnerability in Rspack

```

> Report security bugs in Rspack via web-infra-careers@bytedance.com.

Reviewer: ```suggestion
Report a security vulnerability in Rspack via web-infra-careers@bytedance.com.
```
